### PR TITLE
added ellipsis to the end of the text

### PR DESCRIPTION
**Pull request by bconfortin**
https://github.com/AugurProject/augur/issues/3902

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🌈

 This link will always point to this PR's latest changes: https://peek.run/qo0h6g7

File: packages/augur-ui/src/modules/market-cards/market-card.styles.less [hunk 1]
```diff
@@ -53,12 +53,17 @@
 		grid-column: 2;
 		padding-bottom: @size-16;
 		padding-top: @size-4;
+		overflow: hidden;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.styles.less]: overflow needs to be on the third line of the market title

File: packages/augur-ui/src/modules/market-cards/market-card.styles.less [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	border: 1px solid @color-dark-grey;
 	display: flex;
 	margin: @size-4 0;
-  display: grid;
+  	display: grid;
```
Comment (bthaile) [inline packages/augur-ui/src/modules/market-cards/market-card.styles.less]: don't need to add a space in front. need to format

**Review by bthaile — CHANGES_REQUESTED** [review]
discussed issues in discord. need to have 3 lines with ellipse for really long market titles.

**Review by bthaile — APPROVED** [review]
lgtm

**Pull request merged**